platform/security/certificate/security-certificate-keystoreeditor/pom.xml

```diff
@@ -73,7 +85,9 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            not-yet-commons-ssl
+                            not-yet-commons-ssl,
```

Comment:
We are still using it for only PKCS8 compatibility. We don't use any of the actual SSL code out of it.